website/content/docs/discovery/dns.mdx

```diff
@@ -381,7 +410,7 @@ you can use the following query formats specify namespace but not partition:
 - Specify both namespace and datacenter:
 
   ```text
-  [<tag>.]<service>.service.<namespace>.<datacenter>.<domain>
+  [<tag>.]<service>.service.<namespace>.<datacenter-or-peer>.<domain>
```

Comment:
Intended to support in the future, but no support now since it's always parsed as dc

Reply:
If I understand correctly, this still isn't supported as of 1.14.2 (CC @hashi-derek). That seems fine - this isn't the format we recommend for specifying namespace going forward. I just want to make sure my update is accurate :)